### DISCUSSION TITLE
Manual modes with profiles

### DISCUSSION THREAD

**Discussion by MrXT21** [seerge/g-helper#395] [Ideas]
One thing i use in armoury crate is the manual mode with various profiles. I have a 15w version for silent with custom fan curves to suit the enviroment I sit in so the fans dont start until around 75-80c plus multiple gaming profiles starting at 25w and up to 40w so it would be nice to have something similar in g helper.

**Comment by seerge** [comment]
@MrXT21 you can apply any manual settings (ppts/fans) to all 3 modes via fans + power.

**Reply by MrXT21** [reply]
I know that but i need more than 3 profiles and to not have to edit fan curves and power modes everytime i want a different one.
